ember-app/app/services/board-syncing.js

```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
+
+var BoardSyncingService = Ember.Service.extend({
+
+  //Sync Notifier
+  flashMessages: Ember.inject.service(),
+  syncFlashNotifier: function(){
+    if(this.get('syncInProgress')){
+      this.get('flashMessages').add(this.messageData);
```

Comment:
It occurs to me that `this.messageData` should be cloned lest we modify the original `messageData` object later.